apis/opts/kernel_memory.go

```diff
@@ -0,0 +1,15 @@
+package opts
+
+import units "github.com/docker/go-units"
+
+// ParseKernelMemory parses the kernel memory param of container.
+func ParseKernelMemory(kmem string) (int64, error) {
```

Comment:
Can we get all memory-related functions into one file, xxx/memory.go, there are too many file now

Reply:
good idea.